front/src/modules/activities/components/ActivityComments.tsx

```diff
@@ -74,6 +80,22 @@ export function ActivityComments({ activity }: OwnProps) {
         createdAt: new Date().toISOString(),
       },
       refetchQueries: [getOperationName(GET_ACTIVITY) ?? ''],
+      onCompleted: () => {
+        setTimeout(() => {
+          handleFocus();
+        }, 100);
+      },
+      awaitRefetchQueries: true,
+    });
+  }
+
+  function handleFocus() {
+    const scrollableContainer = document.getElementById(
```

Comment:
is it possible to use a ref instead? this seems to be a non-react way of doing things

Reply:
yeah, we can use a ref, let me update this